iis/mymodule.cpp

```diff
@@ -233,185 +250,17 @@ static void Log(void *obj, int level, char *str)
     mod->WriteEventViewerLog(str, logcat);
 }
 
-#define NOTE_IIS "iis-tx-context"
-
-void StoreIISContext(request_rec *r, RequestStoredContext *rsc)
-{
-    apr_table_setn(r->notes, NOTE_IIS, (const char *)rsc);
-}
-
-RequestStoredContext *RetrieveIISContext(request_rec *r)
-{
-    RequestStoredContext *msr = NULL;
-    request_rec *rx = NULL;
-
-    /* Look in the current request first. */
-    msr = (RequestStoredContext *)apr_table_get(r->notes, NOTE_IIS);
-    if (msr != NULL) {
-        //msr->r = r;
-        return msr;
-    }
-
-    /* If this is a subrequest then look in the main request. */
-    if (r->main != NULL) {
-        msr = (RequestStoredContext *)apr_table_get(r->main->notes, NOTE_IIS);
-        if (msr != NULL) {
-            //msr->r = r;
-            return msr;
-        }
-    }
-
-    /* If the request was redirected then look in the previous requests. */
-    rx = r->prev;
-    while(rx != NULL) {
-        msr = (RequestStoredContext *)apr_table_get(rx->notes, NOTE_IIS);
-        if (msr != NULL) {
-            //msr->r = r;
-            return msr;
-        }
-        rx = rx->prev;
-    }
-
-    return NULL;
-}
-
-
-HRESULT CMyHttpModule::ReadFileChunk(HTTP_DATA_CHUNK *chunk, char *buf)
```

Comment:
Not needed because we are only running in proxy mode? Was it doing any harm? Just thinking it's easier to merge in upstream changes if we don't diverse more than necessary.

Reply:
This one is no longer used as the code processing responses has been removed and it has only been called from there.

I tried to remove all the dead code for keeping it around would pollute the source file for no good reason.